Makefile

```diff
@@ -2,8 +2,8 @@ test:
 	REUSE_DB=1 ./manage.py test
 
 setup:
-	pip install -r requirements.txt
-	pip install django-nose
+	# pip install -r requirements.txt
+	# pip install django-nose
```

Comment:
these should really work fine as part of the setup flow if we want to keep them (and then it would prevent people from having to do them as a separate step). would love to look into it and see if we can figure out what's failing for you